packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/bitmap.dart

```diff
@@ -114,10 +114,10 @@ class BitmapDescriptor {
       _fromAssetImage,
       assetBundleImageKey.name,
       assetBundleImageKey.scale,
-      if (kIsWeb && configuration?.size != null)
+      if (kIsWeb && configuration.size != null)
         [
-          configuration.size.width,
-          configuration.size.height,
+          configuration.size!.width,
+          configuration.size!.height,
```

Comment:
You could make this slightly safer against future changes by making a local variable 
`final size = configuration.size` and then changing all three uses here to `size`. That way the analyzer will know it's not null, and you can eliminate the `!`s.